zip-0311.rst

```diff
@@ -2,7 +2,262 @@
+  non-interactive payment disclosure that reveals the sender's address.
+
+shielded --> shielded
+  The sender, recipient and amount are unknown. Payment disclosure required.
+
+
+Use cases
+=========
+
+Managing contributions for an event
+-----------------------------------
+Alice runs an event on behalf of Bob and Carol, who both agree to split the cost equally.
+Alice receives a single payment for half the amount, and wants proof of who it came from
+(so she knows which person to follow up with for the remaining amount). Carol can provide
+a payment disclosure that reveals:
+
+- The correct amount was sent to the Alice's recipient address in the given transaction.
+- Carol was the sender of that transaction (more precisely, Carol controls the spend
+  authority used in that transaction).
+
+and does not reveal:
+
+- Carol's payment address.
+- Any of Carol's other diversified payment addresses.
+- Any of Carol's other transactions.
+
+Donation drive
+--------------
+
+Diana is a well-known individual with a public Sapling payment address (for receiving
+donations). She runs a matching donation drive to the non-profit Totally Legit Bunnies,
+and wants to prove to her followers that she correctly matched their dontations. Diana
+can create a payment disclosure, and post it to her social media account, that reveals:
+
+- The correct amount was sent to the Totally Legit Bunnies' recipient address in the
+  given transaction.
+- The sender of the transaction is the payment address that is associated with Diana.
+
+and does not reveal:
+
+- Any of Diana's other diversified payment addresses.
+- Any of Diana's other transactions.
+
+Merchant dispute
+----------------
+
+Edward goes to CoinCafe to buy a sandwich, and sends funds from his Sapling address to
+CoinCafe's transparent address. They can see a payment has been made, but claim that they
+have not received a payment from Edward. He can create a payment disclosure that reveals:
+
+- Edward was the sender of the given transaction.
+
+and does not reveal:
+
+- Edward's payment address.
+- Any of Edward's other diversified payment addresses.
+- Any of Edward's other transactions.
+
+Shielded withdrawals from an exchange
+-------------------------------------
+
+CorpBux is an exchange with a transparent hot wallet, and enables customers to withdraw
+their funds to Sapling addresses. They can create a payment disclosure, that they can give
+to their customers, that reveals:
+
+- The correct amount was sent to the customer's recipient address in the given transaction.
+- CorpBux was the sender of the given transaction.
+
+
+Requirements
+============
+
+- The payment disclosure may disclose the contents of zero or more Sapling outputs within
+  a single transaction. (Zero outputs is useful for t2z transactions.)
+- Payment disclosures are non-malleable, and can only be created by a sender of the
+  transaction.
+- A payment disclosure may be tied to a challenge, creating an interactive proof.
```

Comment:
Provide a clearer motivation/example in the Motivation section for which this is desirable.